050-AIS-APIManagementwithFunctions/Student/Challenge-00.md

```diff
@@ -0,0 +1,77 @@
+# Challenge 00 - Preparing your Development Environment
+
+**[Home](../README.md)** - [Next Challenge>](./Challenge-01.md)
+
+## Introduction
+
+A cloud engineer should be have the right set of tools in order to be successful in deploying an integration solution in Azure.
+
+## Description
+
+In this challenge we'll be setting up all the tools we will need to complete our challenges.
+
+  - Install the recommended toolset as follows:
+    - [Azure Subscription](#azure-subscription)
+    - [Azure CLI](#azure-cli)
+    - [Visual Studio Code and Bicep extension](#visual-studio-code-and-bicep-extension)
+      - [Set default Azure subscription and resource group](#set-default-azure-subscription-and-resource-group)
+    - [Managing Cloud Resources](#managing-cloud-resources)
+      - [Azure Portal](#azure-portal)
+      - [Azure Cloud Shell](#azure-cloud-shell)
+
```

Comment:
Since GitHub or ADO is needed, do we want to include that in the recommended toolsets?
Also, since Postman is highly recommended, do we want to include that here as well?

Reply:
Done